### PR TITLE
Update servlist.c

### PR DESCRIPTION
**Pull request by Xandrah**
Added DeltaPool to IRC Networks

### CODE REVIEW

File: src/common/servlist.c [hunk 1]
```diff
@@ -359,6 +359,9 @@ static const struct defaultserver def[] =
 
 	{"Xertion", 0, 0, 0, LOGIN_SASL, 0, TRUE},
 	{0,			"irc.xertion.org"},
+	
+	{"DeltaPool",		0},
```
Comment (TingPing) [inline src/common/servlist.c:363]: Update this to include that it supports SASL and SSL like other entries.

Comment (TingPing) [inline src/common/servlist.c:363]: Update this to include that it supports SASL and SSL like other entries.

**Review by TingPing — CHANGES_REQUESTED** [review]

**Review by Xandrah — COMMENTED** [review]
Updated to support SASL

File: src/common/servlist.c (hunk 1 above)
Comment (Xandrah) [reply src/common/servlist.c]: Done!

**Pull request merged**